app.rb

```diff
@@ -29,7 +29,9 @@ def config
 
     get '/' do
       begin
-        erb :index, locals: { loa1_url: authorization_url(1), loa3_url: authorization_url(3) }
+        erb :index, locals: { loa_url: loa_url,
+                              loa1_link_class: loa1_link_class,
+                              loa3_link_class: loa3_link_class }
```

Comment:
use IAL instead of LOA

Reply:
@amoose 
In the developer docs we refer to LOA1 or LOA3.
https://developers.login.gov/oidc/
We currently display LOA1 and LOA3 next to the sign-in button. Would it be better to create a story to update the docs and the example repos to use IAL instead of LOA?

Reply:
@amoose 
During standup Hooper said that I should go ahead and change LOA to IAL. Working on that now.

Reply:
Done

Reply:
Right. We're doing away with all antiquated language (LOA vs. IAL/AAL). We have a couple stories already to address the 800-63-2 issues in the marketing materials and in our codebases.